### PR TITLE
EOS-27074 - Given a choice to user to build motr rpm using kernel mode or user mode

### PR DESCRIPTION
**Pull request by nikhilpatil2995**
# Problem Statement
- EOS-27074 - Given a choice to user to build motr rpm using kernel mode or user mode

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability. Tested With - with kernel mode - http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/custom-ci-test-EOS-25657/108/console
With User mode - http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/custom-ci-test-EOS-25657/109/console
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: jenkins/custom-ci/centos-7.9.2009/custom-ci.groovy [hunk 1]
```diff
@@ -51,6 +51,12 @@ pipeline {
             description: 'Third Party RPM Version to use.'
         )
 
+        choice(
+            name: 'MOTR_BUILD_MODE',
```
Comment (shailesh-vaidya) [inline jenkins/custom-ci/centos-7.9.2009/custom-ci.groovy:55]: Please make `user-mode` as default. As most of the developers will be using K8s deployment.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: jenkins/custom-ci/centos-7.9.2009/custom-ci.groovy (hunk 1 above)
Comment (nikhilpatil2995) [reply jenkins/custom-ci/centos-7.9.2009/custom-ci.groovy:55]: done

**Review by shailesh-vaidya — APPROVED** [review]

File: jenkins/custom-ci/centos-7.9.2009/motr.groovy [hunk 2]
```diff
@@ -17,6 +17,7 @@ pipeline {
 	parameters {  
         string(name: 'MOTR_URL', defaultValue: 'https://github.com/Seagate/cortx-motr', description: 'Branch for Motr.')
 		string(name: 'MOTR_BRANCH', defaultValue: 'stable', description: 'Branch for Motr.')
+		string(name: 'MOTR_BUILD_MODE', defaultValue: 'user-mode', description: 'Build motr rpm using kernel or user-mode.', trim: true)
```
Comment (gauravchaudhari02) [inline jenkins/custom-ci/centos-7.9.2009/motr.groovy:20]: can't we keep this var in choice format here as well?

Comment (nikhilpatil2995) [reply jenkins/custom-ci/centos-7.9.2009/motr.groovy]: added new changes and tested with  - http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/custom-ci-test-EOS-25657/110/console

**Review by shailesh-vaidya — APPROVED** [review]

**Review by gauravchaudhari02 — APPROVED** [review]

**Pull request merged**